### PR TITLE
updated used-certificate template

### PR DESCRIPTION
**Pull request by cboshae**

### CODE REVIEW

File: used-certificate-email.html [hunk 1]
```diff
@@ -245,49 +245,71 @@
...
-                                                                            {{/ifMatchesRegexStr}}
+                                                                        {{#ifMatchesRegexStr (lower vehicle.provider_name)  "avis"}}
+                                                                        {{else}}
+                                                                        {{#ifMatchesRegexStr dealership.id "77763|19296"}}
+                                                                        {{else}}
+                                                                        {{#if dealership.search_distance}}
+                                                                        <td align="left" valign="middle" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                                        Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                                        font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; color: #333333; line-height: 1.2;
```
Comment (TCdgamez) [inline used-certificate-email.html]: which line height do you want? 1.5 or 1.2?

File: used-certificate-email.html [hunk 2]
```diff
@@ -245,49 +245,71 @@
...
                                                     </table>
                                                 </td>
                                             </tr>
+                                            {{#ifMatchesRegexStr partner.asset_group "chase"}}
+                                            <tr>
+                                                <td style="padding-top: 16px;" class="section-padding" align="center" valign="middle">
+                                                    <table cellpadding="0" cellspacing="0" align="center" style="background: white; border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 450px; margin: 0; padding: 0; border: 2px solid #999999; border-bottom-width: 10px; min-width: 312px" class="responsive-table-480">
+                                                        <tbody>
+                                                            <tr>
+                                                                <td style="padding: 10px 15px; color: #ffffff; background: #999999; font-size: 20px; font-weight: normal; text-align: center" colspan="2">
```
Comment (TCdgamez) [inline used-certificate-email.html]: shorthand for the padding won't work everywhere all the time. If you want it to always show then you need to do `padding-left`, `padding-right`, etc.

File: used-certificate-email.html [hunk 3]
```diff
@@ -245,49 +245,71 @@
...
+                                                                FINANCE YOUR CAR
+                                                                </td>
+                                                            </tr>
+                                                            <tr>
+                                                                <td style="width:50%;padding:15px; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif">
+                                                                    <h2 style="color:#4199d3;font-size:30px;font-weight:normal;margin:10px 0;display:inline-block;">
+                                                                        Apply Online
+                                                                    </h2>
+                                                                    <p style="margin-top: 14px; margin-bottom: 14px; font-size: 14px; line-height: 19px">Get approved and secure financing before heading to your dealer. Confirm your details to start your application.</p>
```
Comment (TCdgamez) [inline used-certificate-email.html]: Margin works eveywhere and the spacing is perfect on all clients with the `<p>`?

File: used-certificate-email.html [hunk 4]
```diff
@@ -245,49 +245,71 @@
...
-                                                                            {{/if}}
-                                                                            {{/ifEq}}
-                                                                            {{/ifEq}}
-                                                                            {{/ifMatchesRegexStr}}
+                                                                        {{#ifMatchesRegexStr (lower vehicle.provider_name)  "avis"}}
+                                                                        {{else}}
+                                                                        {{#ifMatchesRegexStr dealership.id "77763|19296"}}
+                                                                        {{else}}
+                                                                        {{#if dealership.search_distance}}
+                                                                        <td align="left" valign="middle" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
```
[the remark this answers is not in the document]
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 1 above)
Comment (cboshae) [reply used-certificate-email.html]: 1.2

File: used-certificate-email.html [hunk 5]
```diff
@@ -571,56 +625,434 @@ <h2 style="font-size: 22px; font-weight: 200; text-align: left">Your Exclusive C
...
+                                            {{else}}
+                                            {{#ifGt pricing.member_discount 0}}
+                                            <tr>
+                                                <td align="center" valign="middle" class="section-padding">
+                                                    <table width="450" min-width="312" border="0" cellspacing="0" cellPadding="0" align="center" class="responsive-table-480" style="background-color: #ffffff">
+                                                        <tr>
+                                                            <td style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                            Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                            font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center;
+                                                            font-size: 22px; font-weight: 200; padding: 16px" align="center">
```
Comment (TCdgamez) [inline used-certificate-email.html]: Which one is the correct font size?

File: used-certificate-email.html [hunk 6]
```diff
@@ -571,56 +625,434 @@ <h2 style="font-size: 22px; font-weight: 200; text-align: left">Your Exclusive C
...
                                                         <tr>
-                                                            <td colspan="2" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                            <td style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
                                                             Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
-                                                            font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; font-weight:700;
-                                                            padding:24px; text-align: left">
-                                                                Price Details
+                                                            font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center;
+                                                            font-size: 22px; font-weight: 200; padding: 16px" align="center">
```
Comment (TCdgamez) [inline used-certificate-email.html]: two different font-sizes

File: used-certificate-email.html (hunk 1 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html [hunk 7]
```diff
@@ -571,56 +625,434 @@ <h2 style="font-size: 22px; font-weight: 200; text-align: left">Your Exclusive C
...
+                                                    <table width="450" min-width="312" border="0" cellspacing="0" cellPadding="0" align="center" class="responsive-table-480" style="background-color: #ffffff">
                                                         <tr>
-                                                            <td width="75%" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                            <td style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
                                                             Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
                                                             font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center;
-                                                            padding: 0 0 24px 24px; text-align: left" >
-                                                                Advertised Price
+                                                            font-size: 22px; font-weight: 200; padding: 16px" align="center">
```
Comment (TCdgamez) [inline used-certificate-email.html]: I guess find all cases where you have it twice?

File: used-certificate-email.html [hunk 8]
```diff
@@ -245,49 +245,71 @@
                                                                         <td align="left" valign="middle" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
                                                                         Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
                                                                         font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; color: #333333; line-height: 1.2; 
-                                                                        msoLineHeightRule:
-                                                                        exactly; text-align: left">
+                                                                        msoLineHeightRule: exactly; text-align: left">
```
[the remark this answers is not in the document]
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 2 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html [hunk 9]
```diff
@@ -571,56 +625,434 @@ <h2 style="font-size: 22px; font-weight: 200; text-align: left">Your Exclusive C
...
+                                                                    Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                                    font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; font-weight:700;
+                                                                    padding:24px; text-align: left">
+                                                                        Price Details
+                                                                    </td>
+                                                                </tr>
+                                                                <tr>
+                                                                    <td width="75%" style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                                    Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                                    font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center;
```
Comment (TCdgamez) [inline used-certificate-email.html]: text-align: center & left?

Comment (TCdgamez) [reply used-certificate-email.html]: You can just remove the center and it will default to left.

File: used-certificate-email.html [hunk 10]
```diff
@@ -667,90 +1099,494 @@ <h4 style="margin:0">Benefits</h4>
...
+                                            {{#if pricing.member_discount}}
+                                            <tr>
+                                                <td width="100%" align="center" class="section-padding">
+                                                    <table width="450" min-width="312" cellspacing="0" cellPadding="0" class="responsive-table-480" style="border-bottom: 2px solid #e5e5e5; table-layout: fixed">
+                                                        <tr>
+                                                            <td style="color: #343434; font-family: Aktiv Grotesk,HelveticaNeue-Light,
+                                                            Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                            font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; 
+                                                            font-weight:700; padding:24px; text-align:left">
+                                                                <h4 style="margin:0">Benefits</h4>
```
Comment (TCdgamez) [inline used-certificate-email.html]: `h1`-`h6` won't always show the same font-size everywhere. It's flaky. If it's not important you can leave it. I always use `<font>` to set different sizes. Look at my templates for examples.

File: used-certificate-email.html (hunk 5 above)
Comment (cboshae) [reply used-certificate-email.html]: It's always the subsequent styles. In consumer frontend we override styles quite a bit, but they are abstracted by variables. Should have looked closer

File: used-certificate-email.html (hunk 6 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 5 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 7 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 9 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 10 above)
Comment (cboshae) [reply used-certificate-email.html]: Not necessary to change anymore

File: used-certificate-email.html (hunk 3 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html [hunk 11]
```diff
@@ -508,327 +596,1131 @@
...
                                             {{/if}}
-                                            {{#if coupon}}
+                                            {{/if}}
+                                            {{#if coupon.coupon_code}}
                                             <tr>
-                                                <td width="100%" align="center" class="section-padding" style="padding-bottom: 16px">
+                                                <td width="100%" align="center" class="section-padding" style="padding-bottom: 16px;">
                                                     <center>
-                                                        <table width="450" min-width="312" cellspacing="0" cellpadding="0" class="responsive-table-480" style="background-color: #ffffff; table-layout: fixed">
+                                                        <table width="450" min-width="312" cellspacing="0" cellpadding="0" class="responsive-table-480" style="background-color: #ffffff; table-layout: fixed;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: `table-layout: fixed;` probably don't need this here anymore.

File: used-certificate-email.html [hunk 12]
```diff
@@ -508,327 +596,1131 @@
...
-                                                                font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; padding: 24px 24px 0; color: #999999; display: flex; justify-content: center; align-items: center">
-                                                                <img src="{{{mb-logo}}}" alt="Mercedes-Benz Logo" width="33.3%" height="auto" style="max-width: 110px; max-height: 63px; padding-right: 16px"/>
-                                                                <h2 style="font-size: 22px; font-weight: 200; text-align: left">Your Exclusive Control Number</h2>
+                                                                <td style="color: #343434; font-family: 'Aktiv Grotesk',HelveticaNeue-Light,Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                                font-size: 14px; line-height: 1.5; font-weight: 400; text-align: center; padding: 24px 24px 0; color: #999999; display: flex; justify-content: center; align-items: center;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: `display: flex;`? In an email?

File: used-certificate-email.html [hunk 13]
```diff
@@ -508,327 +596,1131 @@
...
+                                                                If you did not receive a Control Number please call <a href="tel:+{{partner.support_phone_number}}" style="text-decoration: none; color: #0098d1;"> {{partner.support_phone_number}} </a> Mon &ndash; Fri, 8am &ndash; 6pm
                                                                 CST to confirm your eligibility for this exclusive member discount.
                                                                 </td>
                                                             </tr>
                                                             <tr>
-                                                                <td align="center" valign="middle" style="padding-bottom: 44px">
-                                                                <table cellpadding="0" cellspacing="0" style="border-collapse: separate">
+                                                                <td align="center" valign="middle" style="padding-bottom: 44px;">
+                                                                <table cellpadding="0" cellspacing="0" style="border-collapse: separate;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: `border-collapse: separate;` is this really needed?

Comment (TCdgamez) [reply used-certificate-email.html]: for the border-radius?

File: used-certificate-email.html [hunk 14]
```diff
@@ -508,327 +596,1131 @@
...
+                                                    </tr>
+                                                {{else}}
+                                                {{#if pricing.transparency_pledge}}
+                                                    <tr>
+                                                        <td width="100%" align="center" class="section-padding">
+                                                            <table width="450" min-width="312" cellspacing="0" cellpadding="0" class="responsive-table-480" 
+                                                            style="border-bottom: 2px solid #e5e5e5; table-layout: fixed;">
+                                                                <tr>
+                                                                    <td colspan="3" style="color: #343434; font-family: 'Aktiv Grotesk',HelveticaNeue-Light,
+                                                                    Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
```
Comment (TCdgamez) [inline used-certificate-email.html]: I honestly hate using colspan with responsive emails but if you can make work its ok. But here it should be `colspan="2"` and the bottom two cells should not have any.

File: used-certificate-email.html [hunk 15]
```diff
@@ -508,327 +596,1131 @@
...
+                                                    </tr>
+                                                {{/if}}
+                                                {{/if}}
+                                            {{else}}
+                                            {{#ifGt pricing.member_discount 0}}
+                                                {{#if vehicle.pricing_flags.show_list_price_post_lead}}
+                                                    <tr>
+                                                        <td width="100%" align="center" class="section-padding">
+                                                            <table width="450" min-width="312" cellspacing="0" cellpadding="0" class="responsive-table-480" 
+                                                            style="border-bottom: 2px solid #e5e5e5; table-layout: fixed;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: same things here as the tables above.

File: used-certificate-email.html [hunk 16]
```diff
@@ -508,327 +596,1131 @@
...
+                                                                </tr>
+                                                            </table>
+                                                        </td>
+                                                    </tr>
+                                                {{else}}
+                                                {{#if pricing.transparency_pledge}}
+                                                    <tr>
+                                                        <td width="100%" align="center" class="section-padding">
+                                                            <table width="450" min-width="312" cellspacing="0" cellpadding="0" class="responsive-table-480" 
+                                                            style="border-bottom: 2px solid #e5e5e5; table-layout: fixed;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: same as above

Comment (TCdgamez) [reply used-certificate-email.html]: omg it keeps going below. you know what to do. 😉

File: used-certificate-email.html [hunk 17]
```diff
@@ -508,327 +596,1131 @@
...
+                                                            {{#if dealership.services.free_scheduled_maintenance_used}}
+                                                            <tr>
+                                                                <td colspan="2" style="color: #343434; font-family: 'Aktiv Grotesk',HelveticaNeue-Light,
+                                                                Helvetica Neue Light,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif;
+                                                                font-size: 14px; line-height: 1.5; font-weight: 400;
+                                                                padding-top: 0px; padding-right: 0px; padding-bottom: 24px; padding-left: 24px;" >
+                                                                    Scheduled Maintenance
+                                                                </td>
+                                                                <td colspan="1" style="color: #343434; font-family: 'Aktiv Grotesk',HelveticaNeue-Light,
+                                                                Helvetica Neue Light,Helvetica Ne
```
Comment (TCdgamez) [inline used-certificate-email.html]: `table-layout: fixed;` I guess try to get rid of all of these. This was only supposed to exist on the paren table.

File: used-certificate-email.html [hunk 18]
```diff
@@ -838,279 +1730,293 @@ <h4 style="margin:0">Dealer Fees</h4>
...
+                                                                            line-height: 1.5; font-weight: 400; text-align: center;
+                                                                            padding-bottom: 15px;">
+                                                                                <a href="mailto:{{dealership.employee.email}}">
+                                                                                    <img src="https://consumer.tcimg.net/assets/18-05/email_button-3665de56b971bfad0609cfe4a6ce14c6.png" width="200" height="auto" style="border: 0;"/>
+                                                                                </a>
+                                                                            </td>
+                                                                            {{/if}}
+                                                                            <td width="20px" class="hide-mobile" style="color: #343434; font-family: 'Aktiv Grotesk',HelveticaNeue-Light,
```
Comment (TCdgamez) [inline used-certificate-email.html]: take off the `px` it should be `width="20"`

File: used-certificate-email.html [hunk 19]
```diff
@@ -1239,57 +2137,57 @@ <h5 style="font-size: 14; font-weight: bold; margin: 0; padding: 0">
...
-                                                                                                    color: #11325f; font-weight: normal; margin: 0 0 10px">
+                                                                                                    color: #11325f; font-weight: normal; margin-top: 0; margin-right: 0; margin-bottom: 10px; margin-left: 0;">
                                                                                                         PROTECT YOUR NEXT VEHICLE
                                                                                                     </td>
                                                                                                 </tr>
                                                                                                 <tr>
                                                                                                     <td>
-                                                                                                        <table style="padding-bottom: 10">
+                                                                                                        <table style="padding-bottom: 10px;">
```
Comment (TCdgamez) [inline used-certificate-email.html]: If you have spacing issues with these tables then you need to do:

`<table border="0" cellspacing="0" cellpadding="0" align="center" style="padding-bottom:10px;">`

File: used-certificate-email.html (hunk 12 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 15 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 16 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 17 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 14 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 18 above)
Comment (cboshae) [reply used-certificate-email.html]: Done

File: used-certificate-email.html (hunk 13 above)
Comment (cboshae) [reply used-certificate-email.html]: Yup its needed for the border-radius

File: used-certificate-email.html (hunk 19 above)
Comment (cboshae) [reply used-certificate-email.html]: Not necessary

**Pull request merged**